components/search/list-renderer/rodent-donor.js

```diff
@@ -22,9 +42,15 @@ export default function RodentDonor({ item: rodentDonor }) {
         <SearchListItemTitle>
           {rodentDonor.strain} {rodentDonor.sex}
         </SearchListItemTitle>
-        <SearchListItemMeta>
-          <div key="lab">{rodentDonor.lab.title}</div>
-        </SearchListItemMeta>
+        {(lab || collections) && (
```

Comment:
Do you also need an `|| phenotypicFeatures` here, or do you not display the meta if you have no lab no collections even if you have phenotypic features?

Reply:
no that's true, I'll add it